examples/model_selection/plot_grid_search_stats.py

```diff
@@ -187,9 +187,11 @@ def corrected_std(differences, n_train, n_test):
 
     Returns
     -------
-    corrected_std : int
+    corrected_std : float
         Variance-corrected standard deviation of the set of differences.
     """
+    # kr = k times r, r times repeated k-fold crossvalidation,
+    # kr equals the number of times the model was fit and evaluated
```

Comment:
```suggestion
    # kr equals the number of times the model was evaluated
```

Reply:
I think this is clearer.

Reply:
Ok. Did I mix up my past tenses, is it "fitted" rather than "fit"? :)